src/modules/reading-room/components/FilterMenu/FilterMenuMobile/FilterMenuMobile.module.scss

```diff
@@ -73,6 +73,10 @@
 	}
 }
 
+.c-filter-menu-mobile__form {
+	height: calc(100% - $c-navigation-height-md);
```

Comment:
Consistent blijven tussen `%` & `vh` als fluid units noodzakelijk zijn

Reply:
`%` is hier nodig, de FilterMenuMobile gaat zelf al met `vh` heel het scherm innemen. Deze styling zorgt ervoor dat hij steeds de volledige hoogte van de parent inneemt. Dit dient er ook voor om de form acties above the fold te houden en de body scrollable als deze te hoog is.